helm-charts/iqserver/README.md

```diff
@@ -0,0 +1,172 @@
+# Sonatype IQ server
+
+[Sonatype Nexus IQ Server](https://www.sonatype.com/nexus-iq-server)  is everything you need to know to trust your software supply chain. It powers Nexus Firewall, Nexus Lifecycle, and Nexus Auditor.
+
+### Prerequisites
+
+- Kubernetes 1.8+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+- Helm 2
+
+### With Open Docker Image
+
+By default, the Chart uses Red Hat's Certified Container. If you want to use the standard docker image, run with `--set iq.imageName=sonatype/nexus-iq-server`.
+
+### With Red Hat Certified container
+
+Red Hat Certified Container (RHCC) requires authentication in order to pull the image. To do this:
+
+  1. [Create a Service Account](https://access.redhat.com/terms-based-registry/)
+  2. Copy the docker configuration JSON sample and replace the host from `registry.redhat.io` to `registry.connect.redhat.com` and save it as a file, eg:
+
+```json
+{
+  "auths": {
+    "registry.connect.redhat.com": {
+      "auth": "TOKEN"
+    }
+  }
+}
+```
+
+If the cluster fails to pull the image, try reverting back to `registry.redhat.io` in the `auths` configuration.
+
+  3. Encode the file in Base 64 format:
+
+```bash
+cat service-auth.json | base64 > service.base64
+```
+  4. Add this to your `myvalues.yaml` as `iq.imagePullSecret`:
+
+```yaml
+iq:
+  name: nxiq
+  imageName: registry.connect.redhat.com/sonatype/nexus-iq-server:1.90.0-01-ubi-1
+  imagePullPolicy: IfNotPresent
+  imagePullSecret: "{BASE64-DOCKER-CONFIG}"
+```
+
+## Initialize Helm/Tiller on the Kubernetes cluster if needed
+
+Install helm/tiller:
+```bash
+$ helm init
+```
+
+## Testing the Chart
+To test the chart:
+```bash
+$ helm install --dry-run --debug ./
+```
+To test the chart with your own values:
+```bash
+$ helm install --dry-run --debug -f my_values.yaml ./
+```
+
+## Installing the Chart
+
+To install the chart:
+
+```bash
+$ helm install ./
```

Comment:
This feels wrong. And should we be talking about installing a 'Chart'?

Reply:
disregard this whole readme from the helm chart.